packages/forms/docs/07-testing.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Testing
+---
+
+All examples in this guide will be written using [Pest](https://pestphp.com). However, you can easily adapt this to a PHPUnit.
+
+Since the form builder works on Livewire components, you can use the [Livewire testing helpers](https://laravel-livewire.com/docs/testing). However, we have custom testing helpers that you can use for forms:
+
+## Forms
+
+To check that a Livewire component has a form, use `assertFormExists`:
+
+```php
+use function Pest\Livewire\livewire;
+
+it('has a form', function () {
+    livewire(CreatePost::class)
+        ->assertFormExists();
+});
+```
+
+> Note that if you have multiple forms on a Livewire component, you can pass the name of a specific form like `assertFormExists('fooForm')`.
+
+## Fields
+
+To ensure that a form has a given field pass the field name to `assertFormFieldExists`:
+
+```php
+use function Pest\Livewire\livewire;
+
+it('has a title field', function () {
+    livewire(CreatePost::class)
+        ->assertFormFieldExists('title');
+});
+```
+
+> Note that if you have multiple forms on a Livewire component, you can specify which form you want to check for the existence of the field like `assertFormFieldExists('title', 'fooForm')`.
+
+### Visibility
+
+To ensure that a field is visible pass the name to `assertFormFieldIsVisible`:
+
+```php
+use function Pest\Livewire\livewire;
+
+test('title is visible', function () {
+    livewire(CreatePost::class)
+        ->assertFormFieldIsVisible('title');
+});
+```
+
+Or to ensure that a field is hidden you can pass the name to `assertFormFieldIsHidden`:
+
+```php
+use function Pest\Livewire\livewire;
+
+test('title is hidden', function () {
+    livewire(CreatePost::class)
+        ->assertFormFieldIsVisible('title');
```

Comment:
Is this correct? I assume you want to show the working of `assertFormFieldIsHidden('title');`

Reply:
Thanks, you're right!